src/app/common/components/CertificateUpload.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { render } from 'react-dom';
+import {
+  Button,
+  InputGroup,
+  TextInput
+} from '@patternfly/react-core';
+
+interface ICertificateUploadProps {
+  isDisabled: boolean;
+  onBlur: any;
+  onChange: any;
+  onInput: any;
+  name: string;
+}
+
+interface ICertificateUploadState {
+  display: string;
+}
+
+class CertificateUpload extends React.Component<ICertificateUploadProps, ICertificateUploadState> {
+
+  private fileInput = React.createRef<HTMLInputElement>();
+
+  constructor(props) {
+    super(props);
+    this.clickFileInput = this.clickFileInput.bind(this);
+    this.state = {
+      display: '',
+    };
+  }
+
+  clickFileInput() {
```

Comment:
change to: 

  `const fileInput = useRef(null);
  const clickFileInput = () =>{
    // redirect a click from any other part of the
    // component to the hidden file input in order
    // to open the browse dialog
    const fileInputCurrent = fileInput.current;
    if (fileInputCurrent) {
      fileInputCurrent.click();
    }
  }

`
`